Repository/EnvironmentRepository.php

```diff
@@ -22,6 +22,17 @@ public function findBy(array $criteria, array $orderBy = null, $limit = null, $o
     	}
     	return parent::findBy($criteria, $orderBy, $limit, $offset);
     }
+    
+    /**
+     * @return array
+     */
+    public function findAllAliases()
+    {
+        $qb = $this->createQueryBuilder('e', 'e.alias');
+        $qb->select('e.alias, e.name, e.managed');
+        

```

Comment:
I'm confuse, does the environment entity as an alias field or an aliases field

Reply:
Yes the environment entity has the following columns: id, name, alias, color, managed, ... The alias is the environment name prefixed with the instance_id parameter.